test/e2e/e2ethanos/services.go

```diff
@@ -38,7 +43,7 @@ var defaultBackoffConfig = util.BackoffConfig{
 
 // TODO(bwplotka): Run against multiple?
 func DefaultPrometheusImage() string {
-	return "quay.io/prometheus/prometheus:v2.19.3"
+	return "quay.io/prometheus/prometheus:main"
```

Comment:
Let's change the image back to `v2.19.3`.
Updating the image to the latest main causes our Rule E2E test broken. Same as https://github.com/thanos-io/thanos/pull/3905#discussion_r591791448.

As I think it is okay to add the E2E test in a later pr, we don't need to update the image now.